### PR TITLE
Added new items DB and corrected scripts (part 4)

### PR DESCRIPTION
**Pull request by zackdreaver**
Please check the scripts before merge.

### CODE REVIEW

File: db/re/item_db.conf [hunk 1]
```diff
@@ -113181,10 +113179,17 @@ item_db: (
 	Buy: 20
 	Weight: 300
 	Def: 2
-	Slots: 1
+	Job: 0x00810204
```
Comment (zackdreaver) [inline db/re/item_db.conf]: Woops, accidently delete the slot

File: db/re/item_db.conf [hunk 2]
```diff
@@ -113490,6 +113675,55 @@ item_db: (
...
+	Buy: 20
+	Weight: 900
+	Def: 6
+	Slots: 1
+	Loc: 256
+	EquipLv: 20
+	View: 899
+	Script: <"
+		bonus2 bAddMonsterDropItem,921,100+((getrefine()/2)*100);
+		bonus2 bAddMonsterDropItem,7033,100+((getrefine()/2)*100);
```
Comment (zackdreaver) [inline db/re/item_db.conf]: Maybe it should be smaller chance

**Comment by zackdreaver** [comment]
@dastgir 
Ok its ready.

**Comment by malufett** [comment]
hmmm..I'm just curious where are you getting your sources? aegis db? or new item descriptions?

**Comment by zackdreaver** [comment]
@malufett
new item descriptions,
mostly i will consult with dastgir first considering the items script.

**Comment by dastgir** [comment]
@malufett, items from description are better than no items...

**Comment by malufett** [comment]
@zackdreaver 
I see..:D

@dastgir 
but I want new scripts..ahahahaha :3

**Comment by MishimaHaruna** [comment]
For item scripts we don't have official data for (i.e. inferred from the item description), I think we should include a comment mentioning it.

**Comment by zackdreaver** [comment]
Ok i will add a comment on items that need it

**Comment by dastgir** [comment]
Thanks for Contribution....

**Pull request merged**